tsc finds recursive reference with TestConfigOverrides

Does the `TestConfigOverrides` need to partial extend itself? I get a tsc recursive reference when checking with `npx tsc` with this package implemented. I tried removing `extends Partial<TestConfigOverrides>` in my local node_modules and it appears to have worked.

https://github.com/javierbrea/cypress-fail-fast/blob/b95ca06e8da9728980ba76277be463598334c43f/index.d.ts#L12

`error TS2310: Type 'TestConfigOverrides' recursively references itself as a base type`

--- a/index.d.ts
+++ b/index.d.ts
@@ -9,7 +9,7 @@ declare namespace Cypress {
     enabled?: boolean
   }
 
-  interface TestConfigOverrides extends Partial<TestConfigOverrides> {
+  interface TestConfigOverrides {
     /**
      * Configuration for fail-fast plugin
     */

